content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -43,55 +40,138 @@ Istio control plane, Envoy can then communicate with the **single**
-    metadata:
-      name: istio-reader
-    rules:
-       - apiGroups: ['']
-         resources: ['nodes', 'pods', 'services', 'endpoints']
-         verbs: ['get', 'watch', 'list']
-    EOF
+    $ helm template install/kubernetes/helm/istio-remote --namespace istio-system --name istio-remote --set global.remotePilotAddress=${PILOT_POD_IP} --set global.remotePolicyAddress=${POLICY_POD_IP} --set global.remoteTelemetryAddress=${TELEMETRY_POD_IP} --set global.proxy.envoyStatsd.enabled=true --set global.proxy.envoyStatsd.host=${STATSD_POD_IP} --set global.remoteZipkinAddress=${ZIPKIN_POD_IP} > $HOME/istio-remote.yaml
+    {{< /text >}}
+
+1.  Create a namespace for remote Istio.
+
+    {{< text bash >}}
+    $ kubectl create ns istio-system
+    {{< /text >}}
+
+1.  Instantiate the remote cluster's connection to the Istio control plane:
+
+    {{< text bash >}}
+    $ kubectl create -f $HOME/istio-remote.yaml
+    {{< /text >}}
+
+1.  Label the remote cluster's `default` namespace for auto-sidecar injection.
+
+    {{< text bash >}}
+    $ kubectl label namespace default istio-injection=enabled
+    {{< /text >}}
+
+    Repeat for any additional kubernetes namespaces to setup auto-sidecar injection.
+
+### Alternatively use Helm and Tiller to connect the remote cluster to the local
+
+1.  If a service account has not already been installed for Helm, please
+install one:
+
+    {{< text bash >}}
+    $ kubectl create -f @install/kubernetes/helm/helm-service-account.yaml@
     {{< /text >}}
 
-1.  Create a `ServiceAccount` named `istio-multi` for the Istio control plane:
+1.  Initialize Helm:
 
     {{< text bash >}}
-    $ export SERVICE_ACCOUNT=istio-multi
-    $ export NAMESPACE=istio-system
-    $ kubectl create ns ${NAMESPACE}
-    $ kubectl create sa ${SERVICE_ACCOUNT} -n ${NAMESPACE}
+    $ helm init --service-account tiller
     {{< /text >}}
 
-1.  Bind the `ServiceAccount` named `istio-multi` to the `ClusterRole` named `istio-reader`:
+1.  Install the Helm chart:
 
     {{< text bash >}}
-    $ kubectl create clusterrolebinding istio-multi --clusterrole=istio-reader --serviceaccount=${NAMESPACE}:${SERVICE_ACCOUNT}
+    $ helm install install/kubernetes/helm/istio-remote --name istio-remote  --namespace istio-system --set global.remotePilotAddress=${PILOT_POD_IP} --set global.remotePolicyAddress=${POLICY_POD_IP} --set global.remoteTelemetryAddress=${TELEMETRY_POD_IP} --set global.proxy.envoyStatsd.enabled=true --set global.proxy.envoyStatsd.host=${STATSD_POD_IP} --set global.remoteZipkinAddress=${ZIPKIN_POD_IP}
     {{< /text >}}
 
+### Helm configuration parameters
+
+> The `pilot`, `policy`, `telemetry`, `statsd`, and tracing application endpoints need to be configured in the `istio-remote` helm chart.
+
+The `istio-remote` Helm chart requires the endpoint specific variables to be configured as defined in the following table.  Optionally, the automatic sidecar injection in the remote cluster can be disabled.
```

Comment:
Optionally, the automatic->Optionally, automatic

Reply:
ok